build.bat

```diff
@@ -427,6 +431,12 @@ if !BUILD_MICRO! == 1 (
 
 	call::solution_compilation_flag_calculation
 
+	if !BUILD_MICRO_24x_COMPATIBILITY! == 1 (
+		set "ADDITIONAL_DEFINES=RTI_MICRO_24x_COMPATIBILITY !ADDITIONAL_DEFINES!"
+	) else (
+		set "rtiddsgen_extra_options=!rtiddsgen_extra_options! -additionalRtiLibraries nddsmetp"
```

Comment:
Is this to support zero copy or flat data? Is this required for shared memory? @franciscoRTI

Reply:
yes, for some reason this is the name that Pro uses; in our codegen templates we translate it to the actual library names that are in Micro. This will be updated to the actual library names here in the branch with micro security support for codegen.